stetl/filters/regexfilter.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
Useful, an example will help, hard to grasp otherwise. Suggestions:

* can't regexes be compiled once during init?
* more uses expected? Maybe a baseclass `RegexFilter` and subclasses `RegexToRecordFilter`?

Reply:
Compilation: good point.
More uses: I haven't thought about it yet. It is possible, but at the moment I don't have any other concrete use cases yet. When looking at the possible formats, I think only struct will be a good option. Although formats like geojson_feature, ogr_feature and etree_element could represent the parsed data, they are too specialized. The output of regexfilter, a dictionary, is not something you would typically write directly.